providers.tf

```diff
@@ -1,13 +1,11 @@
-# This is an example of what a provider looks like.
-#
-# provider "aws" {
-#   alias = "myprovider"
-#   assume_role {
-#     role_arn     = "arn:aws:iam::123456789012:role/MyRole"
-#     session_name = "MySessionName"
-#   }
-#   default_tags {
-#     tags = var.tags
-#   }
-#   region = var.aws_region
-# }
+provider "aws" {
+  # Our primary provider uses our terraform role
```

Comment:
Capitalization:
```suggestion
  # Our primary provider uses our Terraform role
```

Reply:
This should probably get punted up to [cisagov/skeleton-tf-module](https://github.com/cisagov/skeleton-tf-module/blob/d27789b21f87d9d99bf2cbd2ce1093a205105327/examples/basic_usage/main.tf#L2) because this project just pulled the provider configuration from the `basic_usage` example.